tests/socket.rs

```diff
@@ -1265,3 +1265,60 @@ fn header_included() {
     let got = socket.header_included().expect("failed to get value");
     assert_eq!(got, true, "set and get values differ");
 }
+
+#[test]
+#[cfg(all(
+    unix,
+    not(any(
+        target_os = "fuchsia",
+        target_os = "solaris",
+        target_os = "illumos",
+        target_os = "netbsd",
+        target_os = "redox",
```

Comment:
This list should match the list in the code, which currently only include redox. If their is a reason for the exclusion please add a comment.

Reply:
Renamed the test and added a note. We're limited by enabling IP_TOS being generated here